install/kubernetes/Makefile

```diff
@@ -8,7 +8,7 @@ HUBBLE_UI_VERSION := "v0.7.3"
 MANAGED_ETCD_VERSION := "v2.0.7"
 ETCD_VERSION := "v3.4.13"
 NODEINIT_VERSION := "62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8"
```

Comment:
not related to this change, will the same issue might happen for start-script and other images ?

Reply:
I think this node-init image tag refers to the underlying git commit, not the docker manifest sha. So it does not seem affected, *I think*.

Reply:
oh based on the format, I think you are right. I tried to confirm it, but the repo seems empty (or at least for me) https://github.com/cilium/startup-script